administrator/language/de-DE/com_users.ini

```diff
@@ -294,6 +350,9 @@ COM_USERS_OPTION_SELECT_COMPONENT="- Komponente auswählen -"
 COM_USERS_OPTION_SELECT_LEVEL_END="- Endebene auswählen -"
 COM_USERS_OPTION_SELECT_LEVEL_START="- Anfangsebene auswählen -"
 COM_USERS_PASSWORD_RESET_REQUIRED="Passwortzurücksetzung erforderlich"
+COM_USERS_POSTINSTALL_MULTIFACTORAUTH_ACTION="Enable the new Multi-factor Authentication plugins"
+COM_USERS_POSTINSTALL_MULTIFACTORAUTH_BODY="<p>Joomla! comes with a drastically improved <a href="https://en.wikipedia.org/wiki/Multi-factor_authentication" target="_blank" rel="noreferrer">Multi-factor Authentication</a> experience to help you secure the logins of your users.</p><p>Unlike the Two Factor Authentication feature in previous versions of Joomla, users <em>no longer have to enter a Security Code with their username and password</em>. The Multi-factor Authentication happens in a separate step after logging into the site. Until they complete their Multi-factor Authentication validation users cannot navigate to other pages or use the site. This makes Multi-factor Authentication <a href="https://en.wikipedia.org/wiki/Phishing" target="_blank" rel="noreferrer">phishing</a>&ndash;resistant. It also allows for interactive validation methods like WebAuthn (including integration with Windows Hello, Apple TouchID / FaceID and Android Biometric Screen Lock), or sending 6-digit authentication codes by email. Both of these interactive, convenient methods are now available as plugins shipped with Joomla! itself.</p>"
```

Comment:
```suggestion
COM_USERS_POSTINSTALL_MULTIFACTORAUTH_BODY="Joomla! kommt mit einer drastisch verbesserten <a href="https://de.wikipedia.org/wiki/Multi-factor_authentication" target="_blank" rel="noreferrer">Multi-Faktor-Authentifizierung</a>, die es ermöglicht, die Logins der Benutzer abzusichern.</p><p>Anders als bei der Zwei-Faktor-Authentifizierung in früheren Versionen von Joomla müssen die Benutzer <em>nicht mehr einen Sicherheitscode zusammen mit ihrem Benutzernamen und Passwort eingeben</em>. Die Multi-Faktor-Authentifizierung erfolgt in einem separaten Schritt nach der Anmeldung auf der Website. Solange die Validierung der Multi-Faktor-Authentifizierung noch nicht abgeschlossen ist, können die Benutzer nicht zu anderen Seiten wechseln oder die Website nutzen. Dadurch wird die Multi-Faktor-Authentifizierung gegen <a href="https://de.wikipedia.org/wiki/Phishing" target="_blank" rel="noreferrer">Phishing</a> geschützt. Außerdem ermöglicht es interaktive Validierungsmethoden wie WebAuthn (einschließlich der Integration mit Windows Hello, Apple TouchID / FaceID und Android Biometric Screen Lock) oder den Versand von 6-stelligen Authentifizierungscodes per E-Mail. Beide dieser interaktiven, bequemen Methoden sind jetzt als Plugins verfügbar, die mit Joomla! mitgeliefert werden.</p>"
```

Reply:
Im deutschen in den Links immer ' verwenden. Und es scheint der noopener zu fehlen im Core.

Reply:
done

Reply:
> 

see https://github.com/joomla/joomla-cms/pull/38301

Reply:
Leider bin ich nicht fit mit HTML-Code.
Falls noch hier noch ein Fehler ist – könntest du ihn verbessern?

PS: An anderer Stelle habe ich gerade noch im vorletzten Satz das "es" durch ein "sie" getauscht:
... Außerdem ermöglicht **sie** interaktive Validierungsmethoden ...